templates/create-daml-app/ui/src/Credentials.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { encode } from 'jwt-simple';
+import { ledgerId } from './config';
+
+export const APPLICATION_ID: string = 'create-daml-app';
+
+export const SECRET_KEY: string = 'secret';
```

Comment:
I think this deserves a big fat comment that this is only for testing with a link to the docs for auth. Yes it’s in the text for the GTG but people will miss that and only look at the code.

Reply:
Maybe just rename it to `TEST_ONLY_SECRET_KEY`?